editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4129,9 +4129,13 @@ void CanvasItemEditor::_set_anchors_and_offsets_to_keep_ratio() {
 			undo_redo->add_do_method(control, "set_anchor", SIDE_BOTTOM, bottom_right_anchor.y, false, true);
 			undo_redo->add_do_method(control, "set_meta", "_edit_use_anchors_", true);
```

Comment:
What about this?

Reply:
It always stores the value `true`, so I believe it should be left in.